plumbing/format/diff/unified_encoder.go

```diff
@@ -0,0 +1,305 @@
+package diff
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"strings"
+
+	"gopkg.in/src-d/go-git.v4/plumbing"
+)
+
+const (
+	diffInit = "diff --git a/%s b/%s\n"
+
+	chunkInit  = "@@ -%s +%s @@%s\n"
+	chunkCount = "%d,%d"
+
+	noFilePath = "/dev/null"
```

Comment:
This is the way that unified diff uses to represent "no file". Is OS independent.